### DISCUSSION TITLE
IAFD's Actors' profile pics with (?), but it exists on iafd.com

### DISCUSSION THREAD

**Discussion by dimmitri9** [CodyBerenson/PGMA-Modernized#247] [Q&A]
The actor has a profile pic on iafd.com, and with the movie that it's trying to match
![Screenshot 2023-05-25 181752](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/fcd1f7aa-651e-4484-8365-6859f5c67481)
![Screenshot 2023-05-25 181806](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/6f67d379-88cd-4cc1-bea0-d9e5f123d7e9)
![Screenshot 2023-05-25 181815](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/7c490519-cede-4829-b946-c7467ebd0a07)
![Screenshot 2023-05-25 181735](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/27733266-be41-4379-986e-86a2574a27e6)

Don't know why this is happening

Edit:

Well, after trying to refresh metadata, it's all wiped out... =/
![Screenshot 2023-05-25 184710](https://github.com/CodyBerenson/PGMA-Modernized/assets/20586368/ccc1cd35-f530-4005-930e-843c897c1ee3)

**Comment by JPH71** [comment]
This happens because IAFD tries to protect itself from scrapers, there is
code in the scraper that tries to circumvent this - but it does not always
work.
This issue has recently been raised by another user and I will soon start
working on it...

A collection will be created - either IAFD Scrape Error - or Cloud Scraper
Issue and any films thats that get this issue will be placed in this
collection
instead of getting a red x a
�
will be displayed instead....and possibly a different image....

I should have done this by the end of the weekend

Jason xxx
